.ddqa/config.toml

```diff
@@ -216,3 +216,10 @@ jira_issue_type = "Task"
 jira_statuses = ["To Do", "In Progress", "Done"]
 github_team = "agent-release-management"
 github_labels = ["team/agent-release-management"]
+
+[teams."Dynamic Instrumentation"]
+jira_project = "DEBUG"
+jira_issue_type = "Task"
+jira_statuses = ["To Do", "In Progress", "Done"]
+github_team = "debugger-go"
+github_labels = ["team/dynamic-instrumentation"]
```

Comment:
Hey Celian, can you clarify what you mean? Are you referring to how the team name in github is debugger? If so, I need to talk to the rest of my team about this as theres a bunch of things that changing it could break.

Reply:
I actually don't understand the CI failure but it seems to be related to your comment